pkg/shell/cockpit-networking.js

```diff
@@ -2848,10 +2849,11 @@ PageNetworkBridgeSettings.prototype = {
             max_age_input.parents("tr").toggle(options.stp);
         }
 
+        /* TODO: Migrate away from jQuery spaghetti */
```

Comment:
possibly add comment about adding linking code between labels and input controls when this is cleaned up